pkg/errors.go

```diff
@@ -0,0 +1,9 @@
+package nirmataerr
```

Comment:
in general, I prefer if we don't use `nirmata` but name using the function / object involved. For example, `clienterr`?